src/components/joinus/JoinUsHero.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import HeroPlaceholder from "@/assets/joinus/hero_placeholder.jpg";
+import Image from "next/image";
+
+type StaticImageData = {
+  src: string;
+};
+
+export const JoinUsHero: React.FC = () => {
```

Comment:
For consistency sake
```suggestion
function JoinUsHero() {
```